src/Umbraco.Core/Services/LocalizationService.cs

```diff
@@ -84,6 +84,25 @@ public void AddOrUpdateDictionaryValue(IDictionaryItem item, ILanguage? language
     /// <param name="defaultValue"></param>
     /// <returns></returns>
     public IDictionaryItem CreateDictionaryItemWithIdentity(string key, Guid? parentId, string? defaultValue = null)
```

Comment:
Obsolete?

Reply:
Yep - now on the service implementation too, not just the interface